lib/iex/lib/iex/helpers.ex

```diff
@@ -111,8 +112,10 @@ defmodule IEx.Helpers do
         true ->
           []
       end
-    apps |> Enum.reverse |> Enum.each(&Application.stop/1)
-    apps
+    not_started? =
+      Enum.reverse(apps)
```

Comment:
First pipe with bare variable always feels needless to me. :) You know I'm against pipelinemania like this: https://github.com/elixir-lang/ecto/pull/1040/files#diff-062a1f2a531d4698f27d0e1f61db2fbbR529.